hack/operatorhub/templates/csv.tpl

```diff
@@ -10,6 +10,14 @@ metadata:
     description: Run Elasticsearch, Kibana, APM Server, Beats, Enterprise Search, Elastic Agent, Elastic Maps Server and Logstash on Kubernetes and OpenShift
     repository: https://github.com/elastic/cloud-on-k8s
     support: elastic.co
+    operators.openshift.io/valid-subscription: "Basic Elastic license"
```

Comment:
Not sure about the official name of the basic license is it "Elastic Basic license"?

Reply:
Or even Elastic's Basic license. I agree it sounds better. 

@Kushmaro Could you help determine the official name of the basic license?

Reply:
I think "Elastic Basic License" should be good enough to use here